src/pretix/control/templates/pretixcontrol/user/reauth.html

```diff
@@ -8,7 +8,11 @@
         {% csrf_token %}
         <h3>{% trans "Welcome back!" %}</h3>
         <p>
-            {% trans "We just want to make sure it's really you. Please re-enter your password to continue." %}
+            {% if form.backend.url %}
+                {% blocktrans trimmed with verbose_name=form.backend.verbose_name %}We just want to make sure it's really you. Please re-authenticate with '{{ verbose_name}}'.{% endblocktrans %}
```

Comment:
Can you use e.g. ``login_provider`` instead of ``verbose_name`` in the translated string? It's just more helpful to the translaters to understand whats going on :)